assets/js/aplus.js

```diff
@@ -665,28 +669,36 @@ $(function() {
  * to the next
  * Note! Translation requires that at least 'hl' query parameter is copied
  */
-(function(window) {
-    window.addEventListener('click', function(event) {
-        if (
-            event.target.nodeName === 'A'
-            && event.target.href
-            && event.target.protocol === window.location.protocol
-            && event.target.host === window.location.host // hostname:port
-            && (event.target.search === '' || event.target.search === '?') // no search
-        ) {
-            event.preventDefault(); // stop default redirect
```

Comment:
I'm pretty sure this was the line that broke every other JS stuff. Assuming this handler was called fist, then the event stopped here and rest of the handlers were not called. It might be enough not to do the redirect below (`window.location.href = event.target.href`), but keep the editing of the link (`event.target.search = ...`), thus the link content should be edited.

If that doesn't work, because it's possible that the event.target.href is already read into variable, and thus editing it now doesn't change default click action), then I think we can somehow edit the information within the event. In other words, the event handlers could be like this:

1. this function: add `hl=` parameter to the link if some condiftions apply
2. any other event handlers, which may end the event (result: some other action is taken, but the link href will contain `hl=`
3. default click handler in the browser: this should now read the updated event.target.href and redirect with the `hl=`

Reply:
It seems like we want to keep the same logic and code :). But, I already mention some of the problems above. I will use the jquery `on()` method because I am sure that delegation is something we want to have. Besides, the piece of code that assesses the `target="_blank" is also causing some unwanted behaviours in the modal windows.